src/app/app.module.ts

```diff
@@ -1,18 +1,29 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule, FormsModule } from '@angular/forms';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { CreateButtonComponent } from './components/create-button/create-button.component';
+import { AddCardComponent } from './components/add-card/add-card.component';
+import { NewTaskInputDirective } from './directives/new-task-input.directive';
+import { TaskButtonComponent } from './components/task-button/task-button.component';
+
+import { ConstantsService } from 'src/app/services/constants.service';
+import { TaskService } from 'src/app/services/task.service';
+import { UtilityService } from 'src/app/services/utility.service';
 
 @NgModule({
   declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
+    AppComponent,
+    CreateButtonComponent,
+    AddCardComponent,
+    NewTaskInputDirective,
+    TaskButtonComponent
   ],
-  providers: [],
-  bootstrap: [AppComponent]
+  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, FormsModule],
+  providers: [ConstantsService, TaskService, UtilityService],
```

Comment:
Registering all components and services globally might be unnecessary as in larger applications, we don't need need all the services throughout the application. We could create feature-wise modules instead.

Reply:
created feature modules to handle respective components and passed the services accordingly.